src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2669,7 +2693,15 @@ private MemberDeclarationSyntax ParseMemberDeclarationCore(SyntaxKind parentKind
                     MemberDeclarationSyntax result;
                     if (IsMisplacedModifier(modifiers, attributes, type, out result))
                     {
-                        return result;
+                        var misplacedModifier = this.CurrentToken;
```

Comment:
>var misplacedModifier = this.CurrentToken; [](start = 24, length = 42)

It is not clear what is the motivation for this change and why are we not using ```result```.

Reply:
Bad merge during rebase. Fixed.